Agenda-client/src/components/QuizGame/questions.ts

```diff
@@ -31,8 +30,8 @@ const questions = {
       correctAnswer: '2',
       messageForCorrectAnswer: 'Correct answer. Good job.',
       messageForIncorrectAnswer: 'Incorrect answer. Please try again.',
-      explanation:
-        '🤝',
+      // explanation:
```

Comment:
maybe leave a comment here why this is commented out? e.g. TODO implement later. Or if not needed, let's remove it to keep the code clean ✨